pkgs/applications/terminal-emulators/rio/default.nix

```
@@ -111,8 +111,9 @@ rustPlatform.buildRustPackage rec {
       extraArgs = [ "--version-regex" "v([0-9.]+)" ];
     };
 
-    tests = {
-      test = nixosTests.terminal-emulators.rio;
+    tests = lib.optionalAttrs stdenv.isLinux {
```

```suggestion
    tests = lib.optionalAttrs stdenv.buildPlatform.isLinux {
```
There was a (very recent) push to be more specific about what environment's platform is being checked with `isX` calls so cross compilation is at less of a deficit.